### DISCUSSION TITLE
What is the metric used in the RAFT paper?

### DISCUSSION THREAD

**Discussion by srinjoym-cerebras** [ShishirPatil/gorilla#555] [Q&A]
Hey all, great  work . I just wanted to know what is the metric used in the RAFT paper? It was clearly mentioned there.
Thanks.

**Comment by VascoMiguens** [comment]
Have you found the answer for this?@srinjoym-cerebras

**Comment by srinjoym-cerebras** [comment]
@VascoMiguens ,unfortunately No, but i think those are RAG-based metrics, since the paper is about improving the performance of RAG systems.
